src/main/java/com/squareup/javapoet/TypeSpec.java

```diff
@@ -315,15 +317,16 @@ public Builder addEnumConstant(String name, TypeSpec typeSpec) {
       return this;
     }
 
+    public Builder addField(Type type, String name, Modifier... modifiers) {
+      return addField(FieldSpec.builder(type, name, modifiers).build());
+    }
+
+
```

Comment:
Also if you want to get super-crazy nitty I think everywhere else the overload the takes a `-Spec` comes before these convenience versions.